unit1/authentication/test_noop_with_token.py

```
@@ -0,0 +1,143 @@
+            ],
+        },
+    )
+
+    # Call the dependency
+    user_id, username, skip_userid_check, user_token = await dependency(request)
+
+    # Assert the expected values
+    assert user_id == DEFAULT_USER_UID
+    assert username == DEFAULT_USER_NAME
+    assert skip_userid_check is True
+    assert user_token == "spongebob-token"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "user_token"

(S105)